backend/model/top_container.rb

```diff
@@ -306,6 +306,45 @@ def self.bulk_update_container_profile(ids, container_profile_uri)
   end
 
 
+  def self.bulk_update_location(ids, location_uri)
+    out = {:records_updated => ids.length}
+
+    relationship = TopContainer.find_relationship(:top_container_housed_at)
+
+    begin
+      relationship.handle_delete(relationship.find_by_participant_ids(TopContainer, ids).keep_if{|v| v.status == 'current'}.map(&:id))
```

Comment:
Hi @jambun.  I notice here that we're deleting the 'current' location if there is one.  Rather than delete, would they like us to try and retain the container_location audit trail by keeping the relationship and setting the status to 'previous' with an end_date? 

Also might be worth using the non-destructive `Array#select` instead of `keep_if` just in case find_by_participant_ids is ever changed to deliver a cached/shared array.